spec/context_spec.rb

```diff
@@ -157,14 +157,20 @@
     expect(context.outcome).to eq(::LightService::Outcomes::SUCCESS)
   end
 
+  it "can contain false values" do
+    context = LightService::Context.make(:foo => false)
+    expect(context[:foo]).to eq false
+  end
+
   context "when aliases are included via .make" do
     let(:context) do
       LightService::Context.make(
         :foo => "foobar",
+        :foo2 => false,
```

Comment:
The test fails if you change it to `:bar2` and without changing the alias, because (I believe) all keys that match aliases get set to nil.  That behavior is causing a spec failure elsewhere, I'll try to  take a look at it later tonight.